src/julia_gc.c

```diff
@@ -448,11 +457,55 @@ void InitMarkFuncBags(UInt type, TNumMarkFuncBags mark_func)
     TabMarkFuncBags[type] = mark_func;
 }
 
+static inline int JMarkTyped(void * obj, jl_datatype_t * ty)
```

Comment:
OK, that seems fine!

But, if we use this name with `JMark` prefix here now, please let's at least rename `JMarkMPtr` and `JMarkBag` -- e.g. to `JuliaMarkFunctionForMPtrs` and `JuliaMarkFunctionForBags`.

Reply:
I picked `MPtrMarkFunc` and `BagMarkFunc`, respectively. Those should be both clear and distinct enough.